pkg/network/node/pod.go

```diff
@@ -518,7 +518,7 @@ func (m *podManager) setup(req *cniserver.PodRequest) (cnitypes.Result, *running
 		return nil, nil, err
 	}
 
-	if err := m.policy.SetUpPod(podIP.String()); err != nil {
+	if err := m.policy.SetUpPod(v1Pod); err != nil {
 		klog.Errorf("there may be issues with pod isolation: %v", err)
```

Comment:
this error needs to be changed to something like fmt.Errorf("network policy cannot be applied to pod %s (%v)", req.PodName, err)